scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/SchedulerFrontendState.java

```diff
@@ -887,9 +887,10 @@ public synchronized void schedulerStateUpdated(SchedulerEvent eventType) {
 
     @Override
     public synchronized void jobSubmitted(JobState job) {
-        JobState storedJobState = new ClientJobState(job);
+        ClientJobState storedJobState = new ClientJobState(job);
         jobsMap.put(job.getId(), storedJobState);
         sState.getPendingJobs().add(storedJobState);
+        job.setGenericInformation(storedJobState.getGenericInformation());
```

Comment:
Is it not possible to set these generic information earlier, when the JobState instance passed in parameters is created? I am a bit afraid that setting it at this level may not be enough for other cases.